src/main/webapp/app/exercises/shared/structured-grading-criterion/grading-instructions-details/grading-instructions-details.component.ts

```diff
@@ -155,7 +155,7 @@ export class GradingInstructionsDetailsComponent implements OnInit {
         this.markdownEditor.parse();
     }
 
-    hasCriterionCommand(domainCommands: [string, DomainCommand][]): boolean {
+    hasCriterionCommand(domainCommands: [string, DomainCommand | null][]): boolean {
```

Comment:
Do we really need to introduce `null` ?

Reply:
Yes, because the `jhi-markdown-editor` can pass `null` here apparently. I don't know how else we should fix this, and then `instanceof` check makes sure `null` commands are dropped.